gnews_api.py

```diff
@@ -0,0 +1,8 @@
+import os
+import requests
+
+
+def fetch_news(query):
+    api_key = os.getenv('GNEWS_API_KEY')
```

Comment:
Put this in a .env file

Reply:
Sure, I will update the code to load the API key from a .env file using the `python-dotenv` library. This library allows us to load environment variables from a .env file into the os.environ dictionary in Python.